pkg/core/interop_neo.go

```diff
@@ -21,6 +24,23 @@ func storageFind(ic *interop.Context) error {
 		return fmt.Errorf("%T is not a StorageContext", stcInterface)
 	}
 	prefix := ic.VM.Estack().Pop().Bytes()
+	opts := ic.VM.Estack().Pop().BigInt().Int64()
+	if opts&^storage.FindAll != 0 {
+		return fmt.Errorf("%w: unknown flag", errFindInvalidOptions)
+	}
+	if opts&storage.FindKeysOnly != 0 && opts&storage.FindDeserialize != 0 {
```

Comment:
pick0/pick1?

Reply:
It was handled implicitly, but yeah, let it be more readable. Fixed.